_haikus/its_everywhere.md

```diff
@@ -0,0 +1,10 @@
+---
+layout: haiku
+title: Its Everywhere
+author: Abhishek Khandelwal
+---
+Did you see the cloud today?
+They are present everywhere
+Even when you do a simple google seach, 
+check your email or play a simple online game.
```

Comment:
In order to add line breaks to the haikus, you need to have two spaces after each line or use the `<br>` HTML tag.

Haikus follow a syllabic format, so each line should be a certain number of syllables.

Your first line should be 5 syllables, the second line 7 syllables, and the last line 5 syllables.

You can check to make sure you have the right number of syllables by using a tool such as http://www.haikusyllablecounter.com/.

Once you've updated the haiku to follow this format, we can merge it!